docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -6,39 +6,29 @@
 [id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
 = Creating Jira issues for a migration wave
```

Comment:
Chapter 5 of this guide is about creating and configuring a Jira connection. That chapter should be a prereq of this procedure, right?

Reply:
@anarnold97,  seems like Miriam is right :-) Can you confirm?